src/search_algo.hpp

```diff
@@ -367,7 +367,26 @@ loadQuery(GlobalDataHolder<c_indexType, c_origSbjAlph, c_transAlph, c_redAlph, c
         throw QueryException{"Zero sequences submitted."};
     }
 
-    // TODO if constexpr BLASTN mode, reverse complements
+    if constexpr (TGH::blastProgram == seqan::BlastProgram::BLASTN)
+    {
+        //TODO view-ify this at some point. Better yet, move to index.
+        // ATTENTION, this won't work well if qryseqs is concatenated_sequences
```

Comment:
What do we do in this case? Is there a better way then?

Reply:
In that case it would be better to work on the underlying vector directly, because one can pre-allocate the entire thing with one operation.

But this will go away if we stream over the input sequences so I don't think we should worry about it now.